tofu/imas2tofu/_core.py

```diff
@@ -3389,7 +3398,8 @@ def _checkformat_Data_dsig(self, ids=None, dsig=None, data=None, X=None,
 
 
     def to_Data(self, ids=None, dsig=None, data=None, X=None, tlim=None,
-                indch=None, indch_auto=False, Name=None, occ=None, config=None,
+                indch=None, indch_auto=False, Name=None, occ=None,
+                config=None, description_2d=None,
```

Comment:
It would be nice to add a bit of documentation (valid for all other user accessible functions). Even if not all parameters are described, we could try that the new ones we add are documented

Reply:
Your absolutely right,
New commit adds docstr to all main user-accessible methods:
get_data(), to_Plasma2D(), to_Cam(), to_Data() and calc_signal()